### DISCUSSION TITLE
Captcha is impossible

### DISCUSSION THREAD

**Discussion by dopperman** [community/community#113362] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Captcha fails when requesting password resets, encounters a network error. Fortunately I remembered my password and didn't have to rely on it. Quick question: what's the benefit of running the captcha trial 10 times? It seems unlikely that a bot that can crack it nine times in a row will be foiled by the tenth

**Comment by SebastienXia** [comment]
I have write a discussion about it : https://github.com/orgs/community/discussions/111543#discussion-6348019

**Comment by shape55** [comment]
I'm also experiencing this issue.  It's definitely a bug.  See my comment [here](https://github.com/orgs/community/discussions/111543#discussioncomment-8894986).
